app/javascript/controllers/input_controller.tsx

```diff
@@ -0,0 +1,22 @@
+import { ApplicationController } from './application_controller';
+import { httpRequest } from '@utils';
+
+export class InputController extends ApplicationController {
```

Comment:
J'ai l'impression que "InputController", avec une méthode "onInput", ça ne rend pas bien compte que de ce ça permet comme comportement, c'est trop vague.

```suggestion
export class TurboInputController extends ApplicationController {
```